lib/model/model.go

```diff
@@ -1034,13 +1059,41 @@ func (m *model) ClusterConfig(deviceID protocol.DeviceID, cm protocol.ClusterCon
 			l.Infof("Unexpected folder %s sent from device %q; ensure that the folder exists and that this device is selected under \"Share With\" in the folder configuration.", folder.Description(), deviceID)
 			continue
 		}
+
+		var foundRemote, foundLocal bool
+		var remoteDevice, localDevice protocol.Device
```

Comment:
```suggestion
		var remoteDeviceInfo, localDeviceInfo protocol.Device
```

This type (procotol.Device) isn't really awesomely named, confusion with a device ID is easy. :) Mitigating somewhat by variable naming?